### PR TITLE
Feat/use bert as extractive summarizer

### PR DESCRIPTION
**Pull request by yurakuratov**
add extractive summarization model based on BERT

### CODE REVIEW

File: deeppavlov/models/bert/bert_as_summarizer.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        input_type_ids = [f.input_type_ids for f in features]
+        feed_dict = self._build_feed_dict(input_ids, input_masks, input_type_ids)
+        nsp_probs = self.sess.run(self.nsp_probs, feed_dict=feed_dict)
+        return nsp_probs[:, 0]
+
+    def __call__(self, texts: List[str], init_sentences: Optional[List[str]] = None) -> List[List[str]]:
+        """Builds summary for text from `texts`
+
+        Args:
+            texts (List[str]): texts to build summaries for
```
Comment (yoptar) [inline deeppavlov/models/bert/bert_as_summarizer.py]: No need to repeat typing

File: deeppavlov/models/bert/bert_as_summarizer.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+            init_sentences (Optional[List[str]], optional): `init_sentence` is used as the first sentence in summary.
```
Comment (yoptar) [inline deeppavlov/models/bert/bert_as_summarizer.py]:
```suggestion
            init_sentences (Optional[List[str]], optional): ``init_sentence`` is used as the first sentence in summary.
```
everywhere for sphinx rendering

File: deeppavlov/models/bert/bert_as_summarizer.py [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+from deeppavlov.models.preprocessors.bert_preprocessor import BertPreprocessor
+
+logger = getLogger(__name__)
+
+
+@register('bert_as_summarizer')
+class BertAsSummarizer(Component):
```
Comment (yoptar) [inline deeppavlov/models/bert/bert_as_summarizer.py]: `TFModel`?

File: deeppavlov/models/bert/bert_as_summarizer.py [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    def _build_feed_dict(self, input_ids, input_masks, token_types):
+        feed_dict = {
+            self.input_ids_ph: input_ids,
+            self.input_masks_ph: input_masks,
+            self.token_types_ph: token_types,
+        }
+        return feed_dict
+
+    def _get_nsp_predictions(self, sentences: List[str], candidates: List[str]):
+        """Compute NextSentence probability for every (sentence_i, candidate_i) pair.
```
Comment (acriptis) [inline deeppavlov/models/bert/bert_as_summarizer.py]: Is there intentionally the same index i or you mean:
```Compute NextSentence probability for every (sentence_i, candidate_j) pair.```
?

Comment (yurakuratov) [reply deeppavlov/models/bert/bert_as_summarizer.py]: Yes, `(sentence_i, candidate_i)`, you can think of `sentences` and `candidates` as batches.

Comment (yurakuratov) [reply deeppavlov/models/bert/bert_as_summarizer.py]: Do you mean that it is better to remove `every` from docstring?

**Comment by varna9000** [comment]
Can this feature be used with the SlavicBERT model?

**Comment by mu-arkhipov** [comment]
@varna9000 
Try replacing download path  `http://files.deeppavlov.ai/deeppavlov_data/bert/rubert_cased_L-12_H-768_A-12_v2.tar.gz` with `http://files.deeppavlov.ai/deeppavlov_data/bert/bg_cs_pl_ru_cased_L-12_H-768_A-12_v1.tar.gz` and change lines 
```"bert_config_file": "{DOWNLOADS_PATH}/bert_models/rubert_cased_L-12_H-768_A-12_v2/bert_config.json",
"pretrained_bert": "{DOWNLOADS_PATH}/bert_models/rubert_cased_L-12_H-768_A-12_v2/bert_model.ckpt",```
with
```"bert_config_file": "{DOWNLOADS_PATH}/bert_models/bg_cs_pl_ru_cased_L-12_H-768_A-12_v1/bert_config.json",
"pretrained_bert": "{DOWNLOADS_PATH}/bert_models/bg_cs_pl_ru_cased_L-12_H-768_A-12_v1/bert_model.ckpt",```
in the `bert_as_summarizer` configs

**Comment by varna9000** [comment]
@mu-arkhipov I did as you suggested, but I'm not sure what should I expect as outcome. I tried both configs - with initial sentence and the summarisation from start of the document, but the result is just a paragraph of consequent sentences, just like in the original text. There is no summarisation whatsoever.

**Pull request merged**